Gemfile

```diff
@@ -86,14 +86,15 @@ group :development, :test do
   gem 'i18n-tasks'
   gem 'knapsack'
   gem 'pry-byebug'
+  gem 'puma'
```

Comment:
Next step is to get rid of thin, right?

Reply:
Yes, I believe we can remove it from our Gemfile now. I will do that in an upcoming PR that updates other gems. However, we will still see the deprecation error for thin because mailcatcher depends on it: https://github.com/sj26/mailcatcher/blob/master/mailcatcher.gemspec#L38